packages/mdc-chips/_mixins.scss

```diff
@@ -40,9 +42,17 @@
 
 @mixin mdc-chip-ink-color($color) {
   @include mdc-states($color);
+  @include mdc-states-activated($color);
   @include mdc-theme-prop(color, $color);
 }
 
+@mixin mdc-chip-ink-color-activated($color) {
+  &:hover,
+  &--activated {
```

Comment:
I think we normally try to avoid appending class names within mixins, because it doesn't always work intuitively for our clients (especially if they use non-BEM selectors like `#id`).

(Ripple seems to be the only exception to this rule, but I'm not sure why we needed to do it there.)

You could potentially do something like this, but it's kinda gross:

```scss
@mixin mdc-chip-ink-color-activated($color) {
  @if & {
    &:hover,
    &.mdc-chip--activated {
      @include mdc-chip-ink-color($color);
    }
  } @else {
    .mdc-chip:hover,
    .mdc-chip--activated {
      @include mdc-chip-ink-color($color);
    }
  }
}
```

This is one of the reasons I prefer option 3 above: It's a lot cleaner and less complicated, which makes our styles easier to maintain, support, and debug :slightly_smiling_face:

Reply:
I removed the `&--activated`, but I think pseudo-classes (`&:hover`) shouldn't be a problem, right?

Reply:
Right, the pseudo `&:hover` should be fine!